include/tvm/runtime/c_runtime_api.h

```diff
@@ -136,13 +136,16 @@ typedef DLDataType TVMType;
 typedef DLContext TVMContext;
 
 /*!
- * \brief The tensor array stucture to TVM API.
+ * \brief The tensor array structure to TVM API.
  */
 typedef DLTensor TVMArray;
 
 /*! \brief the array handle */
 typedef TVMArray* TVMArrayHandle;
 
+/*! \brief the const array handle */
+typedef const TVMArray* kTVMArrayHandle;
```

Comment:
k prefix in the Google C style refers to constant numbers as opposed to types. so we should not make this change

Reply:
Given that C do not really have const, let us keep the C API as it is

Reply:
Ok. Got it. But I will keep the NDArray's const qualifier. Like ```inline void NDArray::CopyFrom(const DLTensor* other)```